extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/ForwardedServerRequestWrapper.java

```diff
@@ -28,7 +28,7 @@
 import io.vertx.core.net.NetSocket;
 import io.vertx.core.net.SocketAddress;
 
-class ForwardedServerRequestWrapper implements HttpServerRequest, HttpServerRequestInternal {
+class ForwardedServerRequestWrapper extends HttpServerRequestInternal implements HttpServerRequest {
```

Comment:
you can directly extends the new `HttpServerRequestInternal` class in 4.3.5-SNAPSHOT

Reply:
Yes, we should the new wrapper when possible.

Extending these interfaces is leading to a lot of hiccups when updating vertx as it can contain new methods.

Reply:
Thanks, it should be better now